### PR TITLE
HttpController createContext test cases

### PR DESCRIPTION
**Pull request by jochem725**
I've added HttpController test cases, to make sure the API is not changed without a test failure.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/http/HttpController.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ private void createHandlers() {
    * Creates the handlers for a certain camera.
    * @param cam camera to create handlers for.
    */
-  private void createHandlers(Camera cam) {
+  public void createHandlers(Camera cam) {
```
Comment (Kingdorian) [inline backend/src/main/java/com/benine/backend/http/HttpController.java]: If you changed this to public just for testing purposes I think you should create a mock object and use that for testing, that way you can access private methods.

**Comment by Kingdorian** [comment]
Looks good I think it can be merged after the little thing pointed out in the comment is fixed.

**Comment by bvanwijk** [comment]
Good improvements, I agree with merging.

File: backend/src/main/java/com/benine/backend/http/HttpController.java [hunk 2]
```diff
@@ -24,33 +24,46 @@
 
   /**
    * Constructor, creates a new HttpController object.
-   * @param address an internetsocet adress indicating the ports for the server to listen to.
+   * @param address an internetsocet address indicating the ports for the server to listen to.
    * @param logger the logger to use to log to.
    * @param camController the cameracontroller that contains
    *                      the camera's which this server interacts with.
    */
   public HttpController(InetSocketAddress address, Logger logger, CameraController camController) {
```
Comment (Kingdorian) [inline backend/src/main/java/com/benine/backend/http/HttpController.java]: We might want to change this from address to, String adress, int port.

**Comment by Kingdorian** [comment]
Looks good.

**Comment by jochem725** [comment]
@Kingdorian Good point about the address and the port, I've added it.

**Comment by jochem725** [comment]
I've performed a manual merge, please check if it has been done correctly.

**Pull request merged**